src/App.tsx

```diff
@@ -352,17 +353,13 @@ export default function App() {
       return (
         <StyledEngineProvider injectFirst>
           <UserProvider data={{ user, reloadUser }}>
-            <TopBar fullWidth={true}>
-              <TopBarContent
-                organizations={organizations}
-                selectedOrganization={selectedOrganization}
-                setSelectedOrganization={setSelectedOrganization}
-                reloadOrganizationData={reloadData}
-                setShowNavBar={setShowNavBar}
-              />
-            </TopBar>
-            <ToastSnackbar />
-            <NoOrgLandingPage reloadOrganizationData={reloadData} />
+            <OrganizationProvider data={{ selectedOrganization, setSelectedOrganization, organizations, reloadData }}>
```

Comment:
re: the typescript errors, my suggestion:
expose that default value in a constant and use that here (since we won't have a selectedOrganization in this path anyways)